outputs.tf

```diff
@@ -13,6 +13,12 @@ output "heroku_iam_user_id" {
   description = "The ID of the IAM user for Heroku."
 }
 
+output "all_django_vars" {
+  value       = module.heroku.all_config_vars
```

Comment:
These aren't really consistently named with each other, but they are more consistent with the input variables for the modules at each level.